test/e2e/configmap.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("ConfigMap", func() {
 		// Kubelet projects the update into the volume and the container picks
 		// it up. This timeout is based on the default Kubelet sync period (1
 		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 90 * time.Second
+		const podLogTimeout = 150 * time.Second
```

Comment:
If this is an update that needs to be processed by a pod worker, we shouldn't go fine with timeouts. The kubelet needs to wake up and invoke the worker. You can probably engineer a timeout that matches pretty fine with when a pod worker is woken up. But if a worker is already doing work, or many workers are up and competing for cpu, there's very little you can do to guarantee it does work exactly on some edge of the kubelet sync. 

Employ the same strategy as we do for pod readiness? (poll till 5m)